Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -400,9 +407,16 @@ void CGameListCtrl::InsertItemInReportView(long _Index)
 			if (!std::getline(titlestxt, line) && titlestxt.eof())
 				break;
 
-			if (line.substr(0,rISOFile.GetUniqueID().size()) == rISOFile.GetUniqueID())
+			const size_t equals_index = line.find('=');
+			std::string game_id = rISOFile.GetUniqueID();
+
+			// Ignore publisher ID for WAD files
+			if (rISOFile.GetPlatform() == DiscIO::IVolume::WII_WAD)
+				game_id.erase(game_id.size() - 2);
+
+			if (line.substr(0, equals_index - 1) == game_id)
 			{
-				name = StrToWxStr(line.substr(rISOFile.GetUniqueID().size() + 3));
+				name = StrToWxStr(line.substr(equals_index + 2));
```

Comment:
Please strip `'\r'` from the end, e.g. with `StripSpaces()` (`std::getline()` only strips `'\n'`).